src/Microsoft.ML.OnnxTransformer/DnnImageFeaturizerTransform.cs

```diff
@@ -55,7 +53,18 @@ internal DnnImageFeaturizerInput(string outputColumnName, string inputColumnName
     /// | Does this estimator need to look at the data to train its parameters? | No |
     /// | Input column data type | Vector of <xref:System.Single> |
     /// | Output column data type | Vector of <xref:System.Single>, the size of the vector depends on the pre-trained DNN |
+    /// | Required NuGet in addition to Microsoft.ML | Microsoft.ML.OnnxTransformer and Microsoft.ML.OnnxRuntime or Microsoft.ML.OnnxRuntime.Gpu|
```

Comment:
Let's delete this row, and expand it to a section: 

### NuGet requirements. 
* Microsoft.ML.OnnxTransformer
* Microsoft.ML.OnnxRuntime
* Microsoft.ML.OnnxRuntime.Gpu (only if GPU processing is used)
* Each pre-trained DNN model has a separate NuGet that must be included if that model is used:
 ** mention the models with their nuget info